Dockerfile

```diff
@@ -1,16 +1,19 @@
 # Build status-go in a Go builder container
 FROM golang:1.9-alpine as builder
 
+ARG build_tags
+
 RUN apk add --no-cache make gcc musl-dev linux-headers
```

Comment:
needed to change this to
```
RUN apk add --no-cache make gcc musl-dev linux-headers git && \
    go get github.com/prometheus/procfs
```
so that it would build successfully with `BUILD_TAGS="metrics prometheus"`

Reply:
i think it should be in vendor folder

Reply:
Doesn't seem like it is. This is the error I get:
```
Step 6/12 : RUN cd /go/src/github.com/status-im/status-go && make statusgo BUILD_TAGS="$build_tags"
 ---> Running in 9684d2f22f2d
go build -i -o build/bin/statusd -v -tags 'metrics prometheus'  ./cmd/statusd
vendor/github.com/prometheus/client_golang/prometheus/process_collector.go:16:8: cannot find package "github.com/prometheus/procfs" in any of:
	/go/src/github.com/status-im/status-go/vendor/github.com/prometheus/procfs (vendor tree)
	/usr/local/go/src/github.com/prometheus/procfs (from $GOROOT)
	/go/src/github.com/prometheus/procfs (from $GOPATH)
make: *** [Makefile:46: statusgo] Error 1
```

Reply:
I meant that it should be added to vendor directory instead of adding on L6

Reply:
Hm, interesting. I used `dep` to add Prometheus as a dependency. I will take a look at that as it may be missing in `vendor/` actually.